.vscode/launch.json

```diff
@@ -41,7 +41,9 @@
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
-                "--disable-extensions",
+                // Create a temp profile that has no extensions / user settings.
+                // This allows us to only have the C# extension + the dotnet runtime installer extension dependency.
+                "--profile-temp",
```

Comment:
This launches vscode with an empty profile with no user settings / extensions.  This is better than --disable-extensions as it 
1.  Resets settings
2.  Allows us to install dependent extensions if needed.

I would have preferred if we could launch into a specific integration test profile, but that requires the profile to be imported first via a UI wizard.  There's no way to launch and import a profile via the vscode cli that I could find.

See https://github.com/microsoft/vscode/issues/159572#issuecomment-1231859949